From 2a8bb56cf78803c55c1c42213c96b413a9883f58 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Wed, 7 Jul 2021 15:17:15 +0300
Subject: [PATCH 28/29] s32: Add macros for FAT and EXT sdcard partitions

Added macros MMC_PART_FAT and MMC_PART_EXT, corresponding to the FAT and
EXT partitions IDs on the sdcard (i.e. partitions 1 and 2).

The EXT partition will be used in Xen setups to load all the required
binaries instead of the FAT partition, while for the other setups,
the FAT partition will be used.

Upstream-Status: Pending

Issue: ALB-6618
Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 8a22a3a860..2d53c75498 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -605,7 +605,9 @@
 #endif
 
 #define CONFIG_SYS_MMC_ENV_DEV		0
-#define CONFIG_MMC_PART			1
+#define MMC_PART_FAT			1
+#define MMC_PART_EXT			2
+#define CONFIG_MMC_PART			MMC_PART_FAT
 
 #ifdef CONFIG_S32_GEN1
 #define FLASH_SECTOR_SIZE               (64 * 1024) /* 64 KB */
-- 
2.17.1

